pkg/recovery/repair.go

```diff
@@ -48,6 +49,17 @@ func NewRepairHandler(s storage.Storer, logger logging.Logger, pushSyncer pushsy
 	return func(ctx context.Context, m *trojan.Message) error {
 		chAddr := m.Payload
 
+		// check if the chunk exists in the local store and proceed.
+		// otherwise the Get will trigger a unnecessary network retrieve
+		exists, err := s.Has(ctx, swarm.NewAddress(chAddr))
+		if err != nil {
+			return err
+		}
+		if !exists {
+			return fmt.Errorf("chunk repair: chunk not present in local store for repairing")
```

Comment:
maybe would be nice to extract the error to be a `const` or `var` at the beginning of the file?